### DISCUSSION TITLE
更新formily出现form instance from context.

### DISCUSSION THREAD

**Discussion by excxapp** [alibaba/formily#2444] [Q&A 2.x] [answered]
Error: Can not found form instance from context.

`是formliy和antd版本兼容有问题吗
"antd": "^4.16.13",
"@formily/antd": "^2.0.0-rc.20",
 "@formily/core": "^2.0.0-rc.20",`
![image](https://user-images.githubusercontent.com/4355581/142127756-24ec41c9-7816-4fba-a615-b3788433d959.png)

**Comment by janryWang** [comment, marked as answer]
https://github.com/alibaba/formily/discussions/2332
